index.html

```diff
@@ -6030,6 +6035,435 @@ <h2>Required</h2>
+                    "instanceName": "ventilation"
+                  },
+                  {
+                    "rel": "tm:submodel",
+                    "href": "./LED.tm.jsonld",
+                    "type": "application/tm+json",
+                    "instanceName": "led"
+                  }
+                ],
+                "properties" : {
+                    "status" : {"type": "string", "enum": ["On", "Off", "Error"]}
+                }
+              }
+           </pre>
+           <pre class="Ventilation exampleTab1">
+            {
+                "@context": "http://www.w3.org/ns/td",
+                "@type": "tm:ThingModel",
+                "title": "Ventilator Thing Model",
+                "version": {
+                    "model": "1.0.0"
+                },
+                "properties": {
+                    "switch": {
+                        "type": "boolean",
+                        "description": "True=On; False=Off"
+                    },
+                    "adjustRpm": {
+                        "type": "number",
+                        "minimum": 200,
+                        "maximum": 1200
+                    }
+                }
+            }
+           </pre>
+           <pre class="LED exampleTab1">
+            {
+                "@context": "http://www.w3.org/ns/td",
+                "@type": "tm:ThingModel",
+                "title": "LED Thing Model",
+                "version": {
+                    "model": "1.0.0"
+                },
+                "properties": {
+                    "R": {
+                        "type": "number",
+                        "description": "Red color"
+                    },
+                    "G": {
+                        "type": "number",
+                        "description": "Green color"
+                    },
+                    "B": {
+                        "type": "number",
+                        "description": "Blue color"
+                    }
+                },
+                "actions": {
+                    "fadeIn": {
+                        "title": "fadeIn",
+                        "input": {
+                            "type": "number",
+                            "description": "fadeIn in ms"
+                        }
+                    },
+                    "fadeOut": {
+                        "title": "fadeIn",
+                        "input": {
+                            "type": "number",
+                            "description": "fadeOut in ms"
+                        }
+                    }
+                }
+            }
+           </pre>
+         </aside>
+
+
+    <p>
+        Such composition can be introduced by the usage of the <code>links</code> container: 
+        <span class="rfc2119-assertion" id="tm-compose-submodel">
+            A parent <a>Thing Model</a> MUST use at least one <code>links</code> entry with <code>"rel":"tm:submodel"</code> 
```

Comment:
Why is this defined as `MUST`? I am no expert in specifications, but as this is formulated here, this sounds very non-optional to me.
It may rather define an arbitrary amount of links, or do I miss something?

👍 for the `tm:submodel` - much clearer than `subthing`